opentelemetry/proto/agent/common/v1/common.proto

```diff
@@ -49,49 +43,9 @@ message Node {
 // Identifier that uniquely identifies a process within a VM/container.
 message ProcessIdentifier {
```

Comment:
I don't think this is needed as well.

Reply:
Correct, was planning to do in separate PR.